tests/3d_examples/test_3d_elasticSolid_shell_collision/CMakeLists.txt

```diff
@@ -12,12 +12,18 @@ target_sources(${PROJECT_NAME} PRIVATE ${DIR_SRCS})
 target_link_libraries(${PROJECT_NAME} sphinxsys_3d)
 set_target_properties(${PROJECT_NAME} PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY "${EXECUTABLE_OUTPUT_PATH}")
 
-add_test(NAME ${PROJECT_NAME}_particle_relaxation 
-         COMMAND ${PROJECT_NAME} --r=true
-         WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
-add_test(NAME ${PROJECT_NAME} 
-         COMMAND ${PROJECT_NAME} --r=false --i=true
-         WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+        add_test(NAME ${PROJECT_NAME}_particle_relaxation COMMAND ${PROJECT_NAME} --r=true
+                 WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+        add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME} --r=false --i=true
+                 WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+else(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+        file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/run_test.sh
+                DESTINATION ${EXECUTABLE_OUTPUT_PATH})
+        add_test(NAME ${PROJECT_NAME} COMMAND bash ${EXECUTABLE_OUTPUT_PATH}/run_test.sh
+                WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+endif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+
```

Comment:
Platform specific test behavior is undesirable, please keep identical. The least surprising behavior for most people I believe in the first place would be to have the executable running particle generation, particle relaxation, and the time loop if no program options are provided. This makes a single `add_test` needed and scripts superfluous.